src/DIRAC/Core/Base/AgentModule.py

```diff
@@ -154,18 +150,23 @@ def __init__(self, agentName, loadName, baseAgentName=False, properties={}):
         self.__initialized = False
 
     def __getCodeInfo(self):
-        versionVar = "__RCSID__"
-        docVar = "__doc__"
+
         try:
-            self.__agentModule = __import__(self.__class__.__module__, globals(), locals(), versionVar)
+            self.__codeProperties["version"] = importlib.metadata.version(
```

Comment:
probably an extremely stupid question, but isn't it the same as `DIRAC.version` ?

Reply:
`DIRAC.version` is set in `self.__codeProperties["DIRACVersion"]`, this one is supposed to be the "equivalent" of `__RCSID__`